storage_service/locations/models/space.py

```diff
@@ -445,7 +445,7 @@ def move_rsync(self, source, destination, try_mv_local=False):
 
         # Rsync file over
         # TODO Do this asyncronously, with restarting failed attempts
-        command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw', '-r', source, destination]
+        command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw,Da+x', '--perms', '-r', source, destination]
```

Comment:
Thanks for the PR!  I am not familiar with that chmod syntax, is 'Da+x' the same as 'a+X'?

I think this is a good change.  
👍